File: 2 semester/Python labs/lab5/spares/Chassis.py
from spares.Spares import *
from enum_category.SparesCategory import *


class Chassis(Spares):
    category = SparesCategory.RUNNINGGEAR

    def __init__(self, serial_number, price, number_of):
        self.serial_number = serial_number
        self.price = price
        self.number_of = number_of

    def __str__(self):
        return "Serial Number: " + str(self.serial_number) + "  Price: " + str(self.price) + " Category: " + str(self.category.value) + " Number of: " + str(self.number_of)

File: 2 semester/Java labs/Lab_3/src/com/ua/lviv/iot/Chassis.java
package com.ua.lviv.iot;

public class Chassis extends Spares {
  public int numberof;

  public Chassis(int serialNumber, int price, CategoryEnum detailCategory, int numberof/*, String name*/) {
    super(serialNumber, price, detailCategory/*, name*/);
    this.numberof = numberof;
  }

  @Override
  public String toString(){
    return "Serial number: " + getSerialNumber() +
    "  price: " + getPrice() +
    "$  Category: " + getCategory() +
    "  number of detailsl: " + numberof+" pieces";
  }
}
